### PR TITLE
Add support for checking multiple processes

### LINKED ISSUE #3: Check multiple processes
Currently, if you want to check multiple processes, you have to run this multiple times which causes multiple calls to `ps-list`, which subsequently decreases performance. I think it would be useful to have a method that takes an array of processes and returns an array with the ones that exists.

### PR DESCRIPTION
**Pull request by kevva**
Wasn't sure about the method name. Happy to hear your thoughts if you have a better suggestion.

Fixes #3.

### CODE REVIEW

**Comment by sindresorhus** [comment]
I would name it `.all`, like `Promise.all`.

File: readme.md [hunk 1]
```diff
@@ -34,6 +39,16 @@ Type: `number` `string`
 
 Process ID or name to check.
 
+### processExists.array(input)
+
+Returns a `Promise<Array>` with the processes that exists.
```
Comment (sindresorhus) [inline readme.md]: Wonder whether it should return this, an array of booleans, or a Map of input:result. I’m leaning towards the latter. Thoughts?

Comment (sindresorhus) [reply readme.md]: If we add the latter, we could add an additional method called `filterExists` For what you have now.

Comment (kevva) [reply readme.md]: An array of booleans doesn't sound too useful? You wouldn't know what which one would be. I like the latter though!

Comment (sindresorhus) [reply readme.md]: > An array of booleans doesn't sound too useful?

Well, you could match it on the index, but I agree, it's the least useful of the options. Let's go with the latter.

Comment (kevva) [reply readme.md]: > Well, you could match it on the index

Dunno why I thought that wouldn't work. But yes let's go with `Map`.

**Comment by sindresorhus** [comment]
> we could add an additional method called filterExists For what you have now.

Did you want to do this too before a new release? Or should I just go ahead and release now?

**Comment by kevva** [comment]
Oh, forgot to add it to this PR. I'll do a new one :).

**Pull request merged**